admin/src/app/(auth)/osa/page.tsx

```
@@ -0,0 +1,1208 @@
+                            : "#e2e8f0",
+                        backgroundColor: isSelected
+                          ? info.color + "0a"
+                          : isHovered
+                            ? info.color + "05"
+                            : "white",
+                        boxShadow: isSelected ? `0 0 0 1px ${info.color}44` : "none",
+                      }}
+                      onMouseEnter={() => setHoveredEntry(entry.id)}
+                      onMouseLeave={() => setHoveredEntry(null)}
+                      onClick={() => setSelectedEntry(isSelected ? null : entry)}
+                    >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)